src/Mobs/ZombiePigman.cpp

```diff
@@ -51,3 +51,17 @@ void cZombiePigman::KilledBy(TakeDamageInfo & a_TDI)
 
 
 
+
+void cZombiePigman::WriteMetadata(cMetadataWriter & a_Writer) const
+{
+	super::WriteMetadata(a_Writer);
+	a_Writer.WriteBool(IsBaby());  // Is baby
+	// Note: Zombie pigmen inherit zombie metadata
```

Comment:
I'm pretty sure that this one should be plural (you'd say "Pigs inherit animal metadata" and not "Pig inherits animal metadata"), so pigmen makes sense.  (Although you would say `cZombiePigman` inherits `cZombie` metadata, singular, I feel like referring to the entities themselves seems like it should be plural).  If it still seems like this should be singular I can change it.